peekingduck/declarative_loader.py

```diff
@@ -180,6 +178,14 @@ def _edit_config(
                         f"Config for node {node_name} does not have the key: {key}"
                     )
                 else:
+                    if key == "detect_ids" and node_name in (
+                        "model.yolo",
+                        "model.efficientdet",
+                    ):
```

Comment:
I briefly tested and found the mapping to be working after adding `"model.yolox"` here. To echo the comment in the test file, consider adding yolox here since this change is going into node-v1.2.

Also, perhaps it's possible to use something similar to `node_name_list` in `configloader`? Unless it is intentional to hardcode the models here to ensure we've developed/tested the feature.

Reply:
Remove `node_name` verification here and outsource the responsibility to the callee function `self.config_loader.change_class_name_to_id` instead.
This assumes the master map file is updated after a new model has been developed and tested.